doc/configuration/instrumentation.md

```diff
@@ -85,6 +88,19 @@ A prefix to prepend all metric names with before they are sent to the graphite s
 Specifies an environmental variable name from which an additional prefix will be taken.
 In case both `prefix` and `env_prefix` are defined, it will be placed before the `prefix` and separated with a dot.
 
+## Log handler options
+
+### `instrumentation.log.level`
+* **Syntax:** string, one of `"none"`, `"emergency"`, `"alert"`, `"critical"`, `"error"`, `"warning"`, `"notice"`, `"info"`, `"debug"`, `"all"`.
+* **Default:** `"debug"`
+* **Example:** `loglevel = "error"`
+
+Severity level at which all the events will be logged.
```

Comment:
Currently this is true, but just FYI, this can be overridden per event (on setup or even on execution). Just saying... and maybe we could warn about the verbosity of such logs? It's on `debug` for a reason...

Reply:
OK, I've changed the line and added a note on verbosity. I was struggling to phrase it correctly, please take a look if not it's fine.